backend/de.metas.business/src/main/sql/postgresql/system/20-de.metas.business/5622730_sys_gh12393_UpdateInvoiceProcessingServiceCompany.sql

```diff
@@ -0,0 +1,15 @@
+UPDATE c_doctype
+SET docsubtype = 'SI'
+FROM InvoiceProcessingServiceCompany serviceCompanySettings
+WHERE (1 = 1)
+  AND serviceCompanySettings.ServiceInvoice_DocType_ID = c_doctype.c_doctype_id
+  AND serviceCompanySettings.isactive = 'Y'
+  AND c_doctype.docbasetype != 'API';
```

Comment:
hmm, maybe it's a bit early in the morning for me,
but loud thinking,
I think it's the other way around... 
i.e. those invoices are "issued" by the invoice processing company,
so we have to pay them,
so they all are purchase invoices...

Reply:
we can assume that any C_DocType configured in `InvoiceProcessingServiceCompany` is a service processing invoice doctype.
so while i don'T fully understand the `AND c_doctype.docbasetype != 'API';` either, i think we can also ditch it.

Reply:
I might be wrong, but I thought the `c_doctype.docbasetype == 'API'` && `c_doctype.docsubtype` == `null` is the default `Purchase Invoice` doc type.. so the idea was to avoid changing the default `Purchase Invoice` doc type to a `Service fee` one.

But now, I see the query is not ok anyway and it doesn't do any good, so I also think we can drop it